v1/linkml-schemas/evidence.yaml

```diff
@@ -153,6 +154,10 @@ enums:
       published: 
       money_laundering_regulations:
       physical_or_biometric_official:
+  IdentityCheckType:
+    permissible_values:
+      IDENTITY_CHECK:
```

Comment:
We could add some metadata here, just for clarity:

```yaml
  IdentityCheckType:
    permissible_values:
      IDENTITY_CHECK:
        deprecated: true
        comments:
          - Historic value for backward compatibility reasons only.
          - Please use IdentityCheck instead.
      IdentityCheck:
```

Reply:
Not sure what the `deprecated: true` flag will have on the produced JSON schema though.